networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc

```diff
@@ -1,11 +1,56 @@
-[id="learn-about-ovn-kubernetes"]
+[id="about-ovn-kubernetes"]
 = About the OVN-Kubernetes Pod network provider
 include::modules/common-attributes.adoc[]
-:context: learn-about-ovn-kubernetes
+:context: about-ovn-kubernetes
 
 toc::[]
 
-include::modules/nw-ovn-technology-preview.adoc[leveloffset=+1]
+The {product-title} cluster uses a virtualized network for Pod and Service networks.
+The OVN-Kubernetes CNI plug-in is a Pod network provider.
```

Comment:
Do we say "Pod network provider" in other places? "Network plugin" is the standard term (eg, https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/)

Reply:
@danwinship, I can try to track down the PR, but at the end of last year, it was suggested that I use _pod network provider_ instead of SDN. [So I've been referring to these as such](https://docs.openshift.com/aro/4/networking/cluster-network-operator.html#nw-operator-configuration-parameters-for-openshift-sdn_cluster-network-operator). I may have misunderstood the intent of the suggestion, though.

It's not immutable, though, and I can refer to these differently.

I think this sprouted from my mistakenly calling it _OVN-Kubernetes SDN_.